starter_code/index.js

```diff
@@ -1 +1,12 @@
-const Elevator = require('./elevator.js');
+const Elevator = require("./elevator");
+const Person = require("./person");
+
+var e = new Elevator();
+
+let maxence = new Person("Maxence", 1, 4);
+let josh = new Person("Josh", 8, 2);
+
+e.call(maxence);
+e.call(josh);
+
+e._passengersEnter();
```

Comment:
This should be called behind the scenes as part of you logic of the `update` method :)